app/uk/gov/hmrc/fileupload/controllers/FileUploadController.scala

```diff
@@ -65,15 +65,17 @@ class FileUploadController( redirectionFeature: RedirectionFeature,
             if (containsContentType(getFormContentType(formData), contentType, envelopeId)) {
               val file = formData.files.head
               val key = createS3Key(envelopeId, fileId)
-              uploadToQuarantine(key, file.ref.inputStream, file.ref.size).flatMap { uploadResult =>
-                val fileRefId = FileRefId(uploadResult.getVersionId)
-                commandHandler.notify(QuarantineFile(envelopeId, fileId, fileRefId, created = now(), name = file.filename,
-                  contentType = file.contentType.getOrElse(""), file.ref.size, metadata = metadataAsJson(formData)))
-                  .map {
-                    case Xor.Right(_) => Ok
-                    case Xor.Left(e) => Status(e.statusCode)(e.reason)
-                  }
-              }
+              if(file.ref.size != null) {
```

Comment:
lift to `Option` and then `fold` could be more idiomatic than comparison to `null`? not a big deal though